Packages/cdms2/Lib/axis.py

```diff
@@ -735,10 +735,22 @@ def designateTime(self, persistent=0, calendar=None):
             if calendar is not None:
                 self.setCalendar(calendar, persistent)
 
+    # For isTime(), keep track of whether each id is for a time axis or not, for better performance.
+    # This dictionary is a class variable (not a member of any particular instance).
+    idtaxis = {}  # id:type where type is 'T' for time, 'O' for other
+
```

Comment:
maybe we should add a test? See if it's already in and if it is and type changes, throw a warning/error? I don't think it will ever happen but of course now that I said it it is bound to happen...